src/common/paddle/pricing.ts

```diff
@@ -46,10 +46,62 @@ export interface PlusPricingPreview {
   trialPeriod?: TimePeriod | null;
 }
 
+// instead of storing a seed in the DB, having this here makes it easier to modify
+const devPricingMetadata: PlusPricingMetadata[] = [
+  {
```

Comment:
Would seeds not be the same thing, just a JSON file and we have a constant way of updating and setting those?

Reply:
Pretty much the same, I just think it is much harder to update the values manually from the DB than a code editor.

Reply:
but I would argue that in code you have to then keep those changes and watch to not commit them accidentally

Reply:
Also skipping cache in dev mode can mean that you are not aware of cache existing in prod, which can have opposite effect. Can we for now just keep it in the DB and then see how other devs use it and then proceed with development only checks and improvements.